cypress/e2e/cards.cy.ts

```diff
@@ -3,19 +3,68 @@ describe('in my home page', () => {
     cy.visit('http://localhost:4200/');
   });
 
-  it('Finds the "Previous" link', () => {
-    cy.contains('Previous');
+  it('has disabled "Previous" link in the fist page', () => {
+    cy.visit('/');
+    const findInPage = () => {
+      cy.get('li:has(span:contains(Previous))').then((element) => {
+        if (element.hasClass('disabled')) {
+          return;
+        }
+        cy.wrap(element).click();
+        findInPage();
+      });
+    };
+    findInPage();
   });
 
-  it('Finds the "Next" link', () => {
-    cy.contains('Next');
+  it('has disabled "Next" link in the last page', () => {
+    cy.visit('/');
+    const findInPage = () => {
+      cy.get('li:has(span:contains(Next))').then((element) => {
```

Comment:
I would like you to make a habit to DRY your code. 

The only difference between this findInPage and the previous one is the selector.
I would make it reusable just for practice.